### PR TITLE
docs dyn

### PR DESCRIPTION
**Pull request by jd-lara**
Add missing docs for functions useful for dynamic modeling

### CODE REVIEW

File: docs/src/modeler_guide/network_matrices.jl [hunk 1]
```diff
@@ -3,6 +3,76 @@
...
 # [Ybus](https://en.wikipedia.org/wiki/Nodal_admittance_matrix), [PTDF](https://www.powerworld.com/WebHelp/Content/MainDocumentation_HTML/Power_Transfer_Distribution_Factors.htm) and LODF
 
-# Check section [Network Matrices](@ref net_mat)
+# Check section [Network Matrices](@ref net_mat) for more details
 
-# **UNDER CONSTRUCTION**
+# ## Overview
+#
+# Network matrices are implemented in `PowerSystems.jl` as arrays that enable using Branch or Bus
+# names a indexes to facilitate exploration and analysis. Ybus is stored as an SparseMatrix
```
Comment (claytonpbarrows) [inline docs/src/modeler_guide/network_matrices.jl:11]: "a" should be "as"

File: docs/src/modeler_guide/power_flow.jl [hunk 2]
```diff
@@ -2,11 +2,62 @@
 
 # `PowerSystems.jl` provides the capability to run a power flow with the intention of
 # providing a valid initial AC operating point to the system. It is not meant as an analytics
-# tool; the main issue is to determine if the system has feasible AC data.
+# tool; the main issue is to determine if the system has feasible AC data. This solver does
+# not check for reactive power limits or other limiting mechanisms in the grid.
```
Comment (claytonpbarrows) [inline docs/src/modeler_guide/power_flow.jl:6]: This is important and could be more clear. Here are some suggestions:
```
# The power flow tool is not meant for analytics where the principal 
# goal is to determine if the system has settings that produce a feasible AC solution. This power flow routine does
# not check for reactive power limits or other limiting mechanisms in the grid, and can therefore be used to check 
# for solver convergence - making no guarantees of the solution feasibility.

File: docs/src/modeler_guide/power_flow.jl [hunk 3]
```diff
@@ -2,11 +2,62 @@
...
+#    in the static injection devices. Returns a dictionary with results in a DataFrame that
+#    can be exported or manipulated as needed.
+#
+# 2. Solves the power flow and updated the devices in the system to the operating condition.
+#    This model will update the values of magnitudes and angles in the system's buses. It
+#    also updates the active and reactive power flows in the branches and devices connected
+#    to PV buses. This utility is useful to initialize systems before serializing or checking
+#    the addition of new devices is still AC feasible.
+
+# Solving the powwer flow with mode 1:
```
Comment (claytonpbarrows) [inline docs/src/modeler_guide/power_flow.jl:36]: powwer = power

File: docs/src/modeler_guide/power_flow.jl [hunk 4]
```diff
@@ -2,11 +2,62 @@
...
+
+results = solve_powerflow(system_data)
+results["bus_results"]
+
+# Solving the powwer flow with mode 2:
```
Comment (claytonpbarrows) [inline docs/src/modeler_guide/power_flow.jl:41]: powwer = power

File: docs/src/modeler_guide/power_flow.jl [hunk 5]
```diff
@@ -2,11 +2,62 @@
...
+
+# Before running the power flow command this are the values of the
```
Comment (claytonpbarrows) [inline docs/src/modeler_guide/power_flow.jl:43]: this = these

**Review by daniel-thom — APPROVED** [review]

File: docs/src/modeler_guide/power_flow.jl [hunk 6]
```diff
@@ -2,11 +2,62 @@
...
+# voltages:
+
+for b in get_components(Bus, system_data)
+    println("$(get_name(b)) - Magnitude $(get_magnitude(b)) - Angle (rad) $(get_angle(b))")
+end
+
+# [`solve_powerflow!`](@ref) return true or false to signal the successful result of the power
+# flow. This enables the integration of a power flow check into functions. For instance,
```
Comment (claytonpbarrows) [inline docs/src/modeler_guide/power_flow.jl:51]: check into functions such as a function to initialize...

File: docs/src/modeler_guide/power_flow.jl [hunk 7]
```diff
@@ -2,11 +2,62 @@
...
+# initializing dynamic simulations. Also, because [`solve_powerflow!`](@ref) uses
+# [NLsolve.jl](https://github.com/JuliaNLSolvers/NLsolve.jl) all the parameters used for NLsolve
+# are also available for [`solve_powerflow!`](@ref)
+
+solve_powerflow!(system_data; finite_diff = true, method = :newton)
+
+# After running the power flow command this are the values of the
```
Comment (claytonpbarrows) [inline docs/src/modeler_guide/power_flow.jl:58]: this = these

File: src/utils/network_calculations/common.jl [hunk 8]
```diff
@@ -238,3 +238,12 @@ end
 Base.to_index(b::Bus) = get_number(b)
 Base.to_index(b::T) where {T <: ACBranch} = get_name(b)
 Base.to_index(ix::Component...) = to_index.(ix)
+
+"""returns the raw array data of the `PowerNetworkMatrix`"""
+get_data(mat::PowerNetworkMatrix) = mat.data
+
+"""
+    returns the lookup tuple of the `PowerNetworkMatrix`. The first entry corresponds
+    to the first dimension and the second entry corresponds to the second dimension
```
Comment (claytonpbarrows) [inline src/utils/network_calculations/common.jl:247]: is it possible to mention what the dimentions are? e.g 1st is buses, 2nd is branches... 
or is this common func to general to make those claims?

**Review by claytonpbarrows — APPROVED** [review]
LGTM... some typos and wording comments

**Pull request merged**